tests/console/install_otherDE_pattern.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+
+    script_run("zypper lr -d | tee /dev/$serialdev");
+
+    my $pattern   = get_var("DE_PATTERN");
+    my $zypp_type = "pattern";
+    if (check_var("DE_IS_PKG", 1)) {
```

Comment:
I don't think it makes sense to introduce a variable which needs to be set from outside. Why not just set a variable in the tests as this is specific to openSUSE anyway and has nothing to do when which test needs to be triggered. Do I misunderstand?